Framework/Core/Windows/Devices/I2c/I2cDevice.cs

```diff
@@ -32,6 +32,7 @@ public sealed class I2cDevice : IDisposable
         private static object s_deviceLock = new object();

```

Comment:
RE: thread safety on I2C device. I noticed that there is static lock to implement thread safety for the object. Is that a UWP requirement? If not, consider removing, because no other HW type in NETMF is thread safe anyway.